typescript/phoenix-sdk/src/utils/market.ts

```diff
@@ -861,3 +881,220 @@ export function getMarketExpectedOutAmount({
 
   return expectedUnitsReceived;
 }
+
+/**
+ * Get the log authority's address
+ */
+export function getLogAuthorityAddress(): PublicKey {
+  return PublicKey.findProgramAddressSync([Buffer.from("log")], PROGRAM_ID)[0];
+}
+
+/**
+ * Find the trader's seat account on Phoenix market
+ * @param marketPubkey The market's address
+ * @param trader The trader's address
+ */
+export function getSeatAddress(
+  market: PublicKey,
+  trader: PublicKey
+): PublicKey {
+  return PublicKey.findProgramAddressSync(
+    [Buffer.from("seat"), market.toBuffer(), trader.toBuffer()],
+    PROGRAM_ID
+  )[0];
+}
+
+/**
+ * Returns an instruction to place a limit order on a market
+ * Does not check for existence of ATAs or seat (no network calls)
+ * Useful for existing market makers with existing associated token accounts for the market's base and mint tokens
```

Comment:
instead of this line, "Should be used to place limit orders after it's confirmed that the maker has ATAs and a seat on the market"